client/app/services/designer-state.service.js

```diff
@@ -141,12 +105,19 @@
 
       function matchAtomicServiceItemToTabs(srvTemplate) {
         var subTab;
-        var newItem;
         if (srvTemplate.service_type === 'atomic') {
           if (srvTemplate.prov_type !== 'generic') {
             subTab = findSubTabByProvType(srvTemplate.prov_type);
-          } else {
+            if (!subTab) {
+              console.log("Couldn't find toolbox Tab for Prov_Type: " + srvTemplate.prov_type);
```

Comment:
@AllenBW what do you mean by "eslint will throwup otherwise".  From what I see, all passed...so I wonder if we need to adjust travis.